### DISCUSSION TITLE
Maybe a website in the future?

### DISCUSSION THREAD

**Discussion by onyxlee** [SupaGruen/SD-CheatSheet#2] [Ideas]
Could you develop this project into a website, and allows user login and favorite their styles? So you can make some money out of this extremely helpful project as well.
Later, you can even allow users to update and maintain the huge db without your personal effort.

**Comment by atandy** [comment]
@SupaGruen  -- I'd like to chat with you about an idea related to this one. Can we connect?

**Comment by onyxlee** [comment]
Yes, of course, love to contribute and help. My email is ***@***.***,
do you prefer any other messenger apps like Facebook messenger, WhatsApp
and such? Text works too.

**Comment by SupaGruen** [comment]
@onyxlee I just added some basic functionality to like a style/mark it a favorite, feel free to test it -> https://supagruen.github.io/StableDiffusion-CheatSheet/

@atandy I'm a bit introverted, but you can [write me on reddit](https://www.reddit.com/user/dude_nooo/) and I'll take a look

**Reply by atandy** [reply]
@SupaGruen -- no problem! I just sent you a message there :) Cheers!
